ui/base-tool/build.gradle

```diff
@@ -33,7 +33,7 @@ dependencies {
     implementation "org.ccci.gto.android:gto-support-eventbus:${deps.gtoSupport}"
     implementation "org.ccci.gto.android:gto-support-lottie:${deps.gtoSupport}"
     implementation "org.ccci.gto.android:gto-support-material-components:${deps.gtoSupport}"
-    implementation "org.ccci.gto.android:gto-support-picasso:${deps.gtoSupport}"
+    api "org.ccci.gto.android:gto-support-picasso:${deps.gtoSupport}"
```

Comment:
`api` vs `implementation` relates to transitive dependency visibility.

if: A depends on B and B depends on C

if the `C` dependency is defined as an `api` dependency, then `A` can access code out of `C`. if the dependency is an `implementation` dependency, the code in `C` is not visible to `A`.

The main reason for this distinction is to scope code so that you only have direct access to what is a public API vs an internal implementation detail.